### DISCUSSION TITLE
reward consist of in the Safe Velocity task

### DISCUSSION THREAD

**Discussion by needtorch** [PKU-Alignment/safety-gymnasium#125] [Q&A]
Hello! I want know what does the reward consist of in the Safe Velocity task? Is the cost subtracted from the reward after step() returns?
Thank you!

**Comment by muchvo** [comment]
The reward is the same as the original reward in the corresponding task within Gymnasium. The cost is handled independently and will not be deducted from the reward. For more detailed information, please refer to [this section](https://safety-gymnasium.readthedocs.io/en/latest/environments/safe_velocity.html) of the documentation.
